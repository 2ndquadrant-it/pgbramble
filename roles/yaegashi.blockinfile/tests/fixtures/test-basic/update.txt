111 222 333
# BEGIN ANSIBLE MANAGED BLOCK
xxx yyy zzz
XXX YYY ZZZ
# END ANSIBLE MANAGED BLOCK
!!! @@@ ###
