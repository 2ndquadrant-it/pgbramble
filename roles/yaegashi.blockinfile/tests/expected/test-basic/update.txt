111 222 333
# BEGIN ANSIBLE MANAGED BLOCK
aaa bbb ccc
AAA BBB CCC
# END ANSIBLE MANAGED BLOCK
!!! @@@ ###
